README.md

```diff
@@ -328,6 +398,34 @@ $ gem install blueprinter
 
 You should also have `require 'json'` already in your project if you are not using Rails or if you are not using Oj.
 
+## Sorting
+By default the response sorts the keys by name. If you want the fields to be sorted in the order of definition, set the below configuration to true.
+
+Usage:
+
+```ruby
+Blueprinter.configure do |config|
+  config.sort_by_definition = true
```

Comment:
This is probably fine as I don't think we need any more types of sorting, but this may be a little more future-proof/extensible and a little bit better named if it were:
`config.sort_fields_by` where the options could be `[:name_asc, :definition]` with `:name_asc` being the default.